Resources/translations/SonataBlockBundle.hu.xliff

```diff
@@ -22,6 +22,46 @@
                 <source>sonata.block.template</source>
                 <target>Sablon</target>
             </trans-unit>
+            <trans-unit id="sonata.block.ajax">
+                <source>sonata.block.ajax</source>
+                <target>sonata.block.ajax</target>
```

Comment:
Not translated part should not be added. So the translator may fallback to the default language.

Reply:
When did we discuss that? Every someone adds a new key, we asked for this

Reply:
> Every time someone adds a new key, we asked for this

When did you decide that? :-)

No. This is not translated so it should not be added, no need to edit tons of file for nothing.

Plus, this will mask the translation not found warning and it's not good. :wink: